### ISSUE TITLE
Keep Alive Creating Too Many POSTs?

### ISSUE THREAD

**Issue by s-moon** [findsorguk/findsorguk#1220]
Does the KeepAlive process create too much traffic? Seems to be a POST every 5 seconds multiplied by the number of logged on users.

**Comment by portableant** [comment]
You can adjust this via the settings detailed in this guide to that library: 

https://github.com/orangehill/bootstrap-session-timeout 

There can't be that many concurrent users logged in, 200 or so max? Could use the locust python to simulate some load testing on a local machine copy. I ran it with 500 simulated users on my macbook when I scripted this bit in.

**Comment by s-moon** [comment]
Not sure about the numbers of users but the logs are full of references to it which makes viewing them a little more painful without filtering. I don't think it needs to be anywhere as frequent as this tho - I was thinking more like once per minute or two? What do you think?

**Comment by portableant** [comment]
Once per minute would work fine or you could drop it to every 5 or even 15 minutes if need be? Sessions are set to the default 30 minutes.

**Comment by s-moon** [comment]
Yes,15 mins sounds perfect :-) I was also going to stop it going to the /users page because that is interfering with people who are editing records. Can you see any issue with that? Perhaps make the URL be '#' so that it  goes nowhere.
